# ROS message for detected object with YOLO and SORT

std_msgs/Header header
uint8[] ids
std_msgs/Float32MultiArray bboxes
std_msgs/String[] labels